field.js

```diff
@@ -0,0 +1,21 @@
+class Field {
+  td = document.createElement('td');
+
+  btn = document.createElement('button');
+
+  createCell() {
+    this.td.style.border = '1px solid black';
+    this.td.innerHTML = '';
+    this.td.style.width = '55px';
+    this.td.style.height = '55px';
+    return this.td;
+  }
+
+  createButton() {
+    document.body.appendChild(this.btn);
```

Comment:
button should be appended to td element so this method should be invoke by createCell